.cardboardlint.yml

```diff
@@ -0,0 +1,35 @@
+default_directories:
+  - packagename
+default_include:
+  - "*py"
+default_exclude:
+  []
+
+import:
+
+namespace:
+  exclude:
+    - "__init__.py"
+    - "test_*.py"
+
+flake8:
+  config_file:
+
+pylint:
+  default_config_file: pylintrc
```

Comment:
Not necessarily. One of the problems we had with using pylint with chemtools was that different files needed different types of configurations (e.g. code vs tests vs examples). But pylint (from what I didn't find) can't apply different configs for different directories. So we needed to have multiple config files.

The `default_config_file` is the configuration that is used for all the files that have not been explicitly selected. I can change it to `config_file`, but it would still need to be documented somewhere.